<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ViewBug</class>
 <widget class="QMainWindow" name="ViewBug">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>783</width>
    <height>766</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>200</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>View Bug</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <property name="styleSheet">
   <string notr="true">#QLabel{
	
	font: 75 15pt &quot;Arial&quot;;
}

#QPushButton{
	
	font: 75 12pt &quot;Arial&quot;;
	min-width:50px;
	max-width:50px;
	max-height:13px;
	padding: 3px;
}

#QTextEdit{
	max-height:30px;
}</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QFrame" name="viewBug_entireFrameQFrame">
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>500</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <property name="lineWidth">
       <number>0</number>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QLabel" name="viewBug_TitleLabel">
         <property name="text">
          <string>Title</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QTextEdit" name="viewBug_TitleTextEdit">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>300</width>
           <height>30</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="viewBug_DateSubmittedLabel">
         <property name="text">
          <string>Date Submitted:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QTextEdit" name="viewBug_dateSubmittedTextEdit">
         <property name="maximumSize">
          <size>
           <width>300</width>
           <height>30</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="viewBug_IDLabel">
         <property name="text">
          <string>ID:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QTextEdit" name="viewBug_IDTextEdit">
         <property name="maximumSize">
          <size>
           <width>300</width>
           <height>30</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QLabel" name="viewBug_AssignedToLabel">
         <property name="text">
          <string>Assigned To:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QTextEdit" name="viewBug_AssignedToTextEdit">
         <property name="maximumSize">
          <size>
           <width>300</width>
           <height>30</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="viewBug_statusLabel">
         <property name="text">
          <string>Status:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QTextEdit" name="viewBug_StatusTextEdit">
         <property name="maximumSize">
          <size>
           <width>300</width>
           <height>30</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QLabel" name="viewBug_IdentifiedByLabel">
         <property name="text">
          <string>Identified By:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="3">
        <widget class="QTextEdit" name="viewBug_identifiedByTextEdit">
         <property name="maximumSize">
          <size>
           <width>300</width>
           <height>30</height>
          </size>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="viewBug_ApplicationLabel">
         <property name="text">
          <string>Application:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QTextEdit" name="viewBug_ApplicationTextEdit">
         <property name="maximumSize">
          <size>
           <width>300</width>
           <height>30</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QLabel" name="viewBug_PriorityLabel">
         <property name="text">
          <string>Priority:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="3">
        <widget class="QTextEdit" name="viewBug_PriorityTextEdit">
         <property name="maximumSize">
          <size>
           <width>300</width>
           <height>30</height>
          </size>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="viewBug_VersionLabel">
         <property name="text">
          <string>Version:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QTextEdit" name="viewBug_VersionTextEdit_2">
         <property name="maximumSize">
          <size>
           <width>300</width>
           <height>30</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="viewBug_DescriptionLabel">
         <property name="text">
          <string>Description:</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1" colspan="3">
        <widget class="QPlainTextEdit" name="viewBug_DescriptionTextArea">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>1</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>60</height>
          </size>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QPushButton" name="viewBug_ViewMoreButton">
         <property name="text">
          <string>View More</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="7" column="0" colspan="4">
        <widget class="QFrame" name="viewBug_middleFrame">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>300</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="5" column="1">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Add Comment</string>
            </property>
           </widget>
          </item>
          <item row="2" column="2" rowspan="2">
           <widget class="QTableWidget" name="tableWidget">
            <row>
             <property name="text">
              <string>1</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>2</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>3</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>4</string>
             </property>
            </row>
            <column>
             <property name="text">
              <string>Filename</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Date Added</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Download</string>
             </property>
            </column>
           </widget>
          </item>
          <item row="4" column="4">
           <widget class="QPlainTextEdit" name="viewBug_HistoryTextArea">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="5" column="2">
           <widget class="QTextEdit" name="textEdit_3"/>
          </item>
          <item row="4" column="2">
           <widget class="QPlainTextEdit" name="viewBug_CommentTextArea">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QLabel" name="viewBug_CommentLabel">
            <property name="text">
             <string>Comments</string>
            </property>
           </widget>
          </item>
          <item row="3" column="4">
           <widget class="QPushButton" name="viewBug_addToCCListButton">
            <property name="text">
             <string>Add To CC List</string>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QLabel" name="viewBug_CCListLabel">
            <property name="text">
             <string>CC List</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QTextEdit" name="viewBug_SeverityTextEdit">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>30</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLabel" name="viewBug_SeverityLabel">
            <property name="text">
             <string>Severity:</string>
            </property>
           </widget>
          </item>
          <item row="7" column="2" rowspan="2">
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QPushButton" name="viewBug_addCommentButton">
              <property name="text">
               <string>Add Comment</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="addBug_uploadFile">
              <property name="text">
               <string>Upload File</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="0" column="4" rowspan="3">
           <widget class="QPlainTextEdit" name="viewBug_CCListTextArea">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QTextEdit" name="viewBug_PlatformTextEdit">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>30</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="viewBug_platformLabel">
            <property name="text">
             <string>Platform:</string>
            </property>
           </widget>
          </item>
          <item row="4" column="3">
           <widget class="QLabel" name="viewBug_HistoryLabel">
            <property name="text">
             <string>History</string>
            </property>
           </widget>
          </item>
          <item row="6" column="2">
           <widget class="QComboBox" name="comboBox"/>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>783</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
